index.d.ts

```diff
@@ -17,9 +18,7 @@ export type navigateTo = (
     cb?: () => Page,
 ) => Promise<Page>;
 
-export interface ModalOptions {
-    context?: any;
-    fullscreen?: boolean;
+export interface ModalOptions extends Partial<ShowModalOptions> {
     props?: Record<string, any>;
```

Comment:
Can you add `target` to the interface? Introduced in this commit: https://github.com/nativescript-vue/nativescript-vue/commit/31bc425c1b9827b1716a0e48a6254798f62bfe76

I think we can type it as `any`.